tests/lookup_test.py

```diff
@@ -186,3 +186,19 @@ def test_lookup_2d_coord():
     )
 
     assert sc.identical(sc.lookup(da, dim='y')[edges + 0.1], expected, equal_nan=True)
+
+
+def test_lookup_2d_coord_outer_dim():
+    edges = sc.array(
+        dims=['x', 'y'], values=[[0.0, 1.0, 2.0], [1.0, 2.0, 3.0]]
+    ).transpose()
+    da = sc.DataArray(
+        sc.array(dims=['x', 'y'], values=[[10.0, 11.0], [12.0, 13.0]]).transpose(),
+        coords={'y': edges},
```

Comment:
I think this was already working for transposed *data*, but we should check for `y` being the outer dim in the *edges* (which is the new case that this fix supports).

Reply:
Isn't that what the `transpose()` on the `edges` is doing? Or did I miss something?

Reply:
Sorry, I did not see that. But you should `transpose().copy()` as otherwise you have not changed the strides, which may cause trouble here.

Reply:
You're correct that it fails with `transpose().copy()`.